k8s-install/scripts/install-cni.sh

```diff
@@ -191,14 +191,16 @@ while [ "$should_sleep" == "true"  ]; do
 	# version to the host. Just check the timestamp on the certificate to see if it
 	# has been updated.  A bit hokey, but likely good enough.
 	if [ "$(ls ${SECRETS_MOUNT_DIR} 2>/dev/null)" ];
```

Comment:
I think you should just change this line instead of adding a new check.  I don't think there is a need for 2 checks.